AsyncDisplayKit/ASTextNode.mm

```diff
@@ -470,6 +485,8 @@ - (id)_linkAttributeValueAtPoint:(CGPoint)point
    inAdditionalTruncationMessage:(out BOOL *)inAdditionalTruncationMessageOut
                  forHighlighting:(BOOL)highlighting
 {
+  ASDisplayNodeAssertMainThread();
```

Comment:
Yep accessing the `_renderer` without holding a lock does not makes sense. I will let it in for now until we have a better naming scheme and kind of similar infrastructure as @ocrickard recommended, that btw I really like.